services/galley/src/Galley/API/TeamNotifications.hs

```diff
@@ -0,0 +1,91 @@
+{-# LANGUAGE StrictData #-}
+
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2020 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+-- | This module is a work-around for the fact that we do not send notifications to all users
+-- of large teams any more.  For greeting bots to hear from new users, the 'MemberJoin' event
+-- is stored in a queue that every member of a team can pull, in a similar (but not identical)
+-- way as gundeck's @/notifications@ end-point, galley has a @/teams/notifications@ end-point
+-- where these events can be pulled.
+--
+-- The module is a clone of "Gundeck.Notification".
```

Comment:
```suggestion

-- | This module is a clone of "Gundeck.Notification".
```

1. the comment is now available in Haddock documentation
2. the empty line separates the license header from the comment, allowing our license header management tool to do its job

Reply:
I did the `This`, but the `|` is already up there in the haddocks that are folded away.

ecc053fc29f040c1b72b0efacfc827d89c0cb2d5